docs/content/docs/libs/state_processor_api.md

```diff
@@ -514,3 +512,119 @@ savepointWriter
+    myAccountValueState = getRuntimeContext().getState(new ValueStateDescriptor<>("MyAccountValueState", Account.class));
+    myValueState = getRuntimeContext().getListState(new ListStateDescriptor<>("MyListState", Integer.class));
+    myMapState = getRuntimeContext().getMapState(new MapStateDescriptor<>("MyMapState", Integer.class, Integer.class));
+  }
+  ...
+}
+```
+
+Then it can read by querying a table created using the following SQL statement:
+```SQL
+CREATE TABLE state_table (
+  k INTEGER,
+  MyValueState INTEGER,
+  MyAccountValueState ROW<id INTEGER, amount DOUBLE>,
+  MyListState ARRAY<INTEGER>,
+  MyMapState MAP<INTEGER, INTEGER>,
+  PRIMARY KEY (k) NOT ENFORCED
+) WITH (
+  'connector' = 'savepoint',
+  'state.backend.type' = 'rocksdb',
+  'state.path' = '/root/dir/of/checkpoint-data/chk-1',
+  'operator.uid' = 'my-uid'
+);
+```
+
+### Connector options
+
+#### General options
+| Option             | Required | Default | Type                                          | Description                                                                                                                                                                                       |
+|--------------------|----------|---------|-----------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| connector          | required | (none)  | String                                        | Specify what connector to use, here should be 'state'.                                                                                                                                            |
+| state.backend.type | required | (none)  | Enum Possible values: hashmap, rocksdb, forst | Defines the state backend which must be used for state reading. This must match with the value which was defined in Flink job which created the savepoint or checkpoint.                          |
+| state.path         | required | (none)  | String                                        | Defines the state path which must be used for state reading. All file system that are supported by Flink can be used here.                                                                        |
+| operator.uid       | optional | (none)  | String                                        | Defines the operator UID which must be used for state reading (can't be used together with `operator.uid.hash`).                                                                                  |
+| operator.uid.hash  | optional | (none)  | String                                        | Defines the operator UID hash which must be used for state reading (can't be used together with `operator.uid`).                                                                                  |
```

Comment:
Additional note that either `operator.uid` or `operator.uid.hash` must be specified?

Reply:
Added.